src/Chapter1.hs

```diff
@@ -508,9 +507,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
-
-
+closestToZero x y = if abs x < (abs y) then x else y
```

Comment:
`()` are redundant in here, so you can go ahead and remove them for cleaner code :slightly_smiling_face: 
```suggestion
closestToZero x y = if abs x < abs y then x else y
```